src/wallet/wallet.h

```diff
@@ -414,7 +414,12 @@ class CWalletTx
         }
 
         ReadOrderPos(nOrderPos, mapValue);
-        nTimeSmart = mapValue.count("timesmart") ? (unsigned int)atoi64(mapValue["timesmart"]) : 0;
+        uint64_t temp;
+        if (mapValue.count("timesmart") && ParseUInt64(mapValue["timesmart"], &temp)) {
```

Comment:
In commit "refactor: Replace use of atoi64 in timesmart unserialize" (f4218c26bbbcfc74ae23db9f9a9294447a8e9a6a)

Note: Slightly stricter parsing is done here (no whitespace or trailing characters allowed) but this should be fine since string just comes from calling `strprintf("%u")`. Could potentially log or return an error though if parsing fails, since it would be an indication of corruption.